src/FqnUtilities.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace LanguageServer;
+
+use phpDocumentor\Reflection\{Type, Types};
+use Microsoft\PhpParser;
+
+class FqnUtilities
+{
+    /**
+     * Returns all possible FQNs in a type
+     *
+     * @param Type|null $type
+     * @return string[]
+     */
+    public static function getFqnsFromType($type): array
```

Comment:
Only one I can think of is that to use a file of pure functions from another namespace, you have to import them individually. Is there a way around that?

https://github.com/felixfbecker/php-language-server/blob/master/src/FilesFinder/FileSystemFilesFinder.php#L9

Putting them in a class gives you one thing to import?